src/main/scala/nodes/learning/GaussianMixtureModelEstimator.scala

```diff
@@ -110,10 +110,8 @@ case class GaussianMixtureModelEstimator(
     while ((iter < maxIterations) && costImproving && largeEnoughClusters) {
       /* E-STEP */
 
-      /*
-      compute the squared malhanobis distance for each gaussian.
-      sq_mal_dist(i,j) || x_i - mu_j||_Lambda^2.
-      */
+      // compute the squared malhanobis distance for each gaussian.
+      // sq_mal_dist(i,j) || x_i - mu_j||_Lambda^2.
       val sqMahlist = (XSq * gmmVars.map(0.5 / _).t) - (X * (gmmMeans :/ gmmVars).t) +
           (DenseMatrix.ones[Double](numSamples, 1) * (sum(gmmMeans :* gmmMeans :/ gmmVars, Axis._1).t :* 0.5))
```

Comment:
Done - I split it into two lines and did a in-place sum using breeze's broadcasting.